Update javascript-client.md (#34182)
dotnet/AspNetCore.Docs: aspnetcore/signalr/javascript-client.md

[!code-html[](javascript-client/samples/6.x/SignalRChat/Pages/Index.cshtml?name=snippet_CDN)]

> In the preceding markup, version 6.0.1 is specified. Use the  following CDNs to get the latest client library version:

Author: ```suggestion
In the preceding markup, version 6.0.1 is specified. Use one of the following CDNs to get the latest client library version:
```

> In the preceding markup, version 6.0.1 is specified. Use the  following CDNs to get the appropriate client library version:

Author: ```suggestion
In the preceding markup, version 6.0.1 is specified. Use the  following CDNs to get the latest client library version:
```

> In the preceding markup, version 6.0.1 is specified. Use one of the following CDNs to get the latest client library version:

Author: ```suggestion
In the preceding markup, version 6.0.1 is specified. Chose from one of the following CDNs to get the latest client library version:
```

> In the preceding markup, version 6.0.1 is specified. Chose from one of the following CDNs to get the latest client library version:

Author: ```suggestion
In the preceding markup, version 6.0.1 is specified. Choose from one of the following CDNs to get the latest client library version:
```
